crdt/or-set.go

```diff
@@ -185,6 +183,10 @@ func (s *ORSet) WriteORSetToFile(needsLocking bool) error {
 // of a supplied ORSet.
 func (s *ORSet) GetAllValues() []string {
 
+	// Read-lock set and unlock on exit.
+	s.lock.RLock()
+	defer s.lock.RUnlock()
```

Comment:
Valid addition if this function should be used in any non-initialization environment. The reason this function did not feature such lock up until now was that it is only ever used in node initialization functions which possess exclusive access to the structure. But I agree, this is a valid safeguard.